src/kv/tx.h

```diff
@@ -254,11 +255,11 @@ namespace kv
 
       for (auto domain_it : grouped_maps)
       {
-        for (auto curr_map : domain_it.second)
+        for (auto curr_view : domain_it.second)
```

Comment:
As `get_maps_grouped_by_domain` returns a `std::map<kv::SecurityDomain, std::vector<AbstractMap*>>`, I would have thought that `curr_map` was correct?

Reply:
This is, if I'm reading it right, `std::map<kv::SecurityDomain, std::vector<AbstractTxView*>>`. So its a collection of views, not maps. There are 2 functions called `get_maps_grouped_by_domain`, and only one is called? I'll rename this (and related variales) to `get_VIEWs_...`, and remove the unused one.